README.md

```diff
@@ -1,3 +1,5 @@
 # chimps
 
-`gulp`
\ No newline at end of file
+Get node dependencies: `npm install`
+Run build process: `gulp`
```

Comment:
`npm start` is preferable as it will run the gulp version specified in `package.json` and installed in the `node_modules` folder. Running `gulp` directly on the command line will look for a globally installed gulp package and will use that if found.